dockerfiles/agents/Dockerfile.author.agent

```diff
@@ -0,0 +1,18 @@
+FROM bcgovimages/von-image:py36-1.16-1
+
+USER root
+RUN apt-get update
+RUN apt-get update && apt-get install -y gcc
+
+RUN pip3 install aries-cloudagent[askar,indy]==0.7.4
```

Comment:
Can we also use the `aries-cloudagent:py36-1.16-1_0.7.4` as base here? Would be good to have a consistent docker setup